lib/rspec_api_documentation/configuration.rb

```diff
@@ -118,7 +118,9 @@ def self.add_setting(name, opts = {})
     # See RspecApiDocumentation::DSL::Endpoint#do_request
     add_setting :response_body_formatter, default: Proc.new { |_, _|
       Proc.new do |content_type, response_body|
-        if content_type =~ /application\/.*json/
+        if response_body.encoding == Encoding::ASCII_8BIT
+          "[binary data]"
+        elsif content_type =~ /application\/.*json/
```

Comment:
presumably this intended to match all json types, such as application/vnd.api+json